README.md

```diff
@@ -13,7 +13,7 @@ in Java, configure a network of nodes and run the service in it.
 For more information see the relevant documentation on the tool:
   - [What is Exonum?](https://exonum.com/doc/get-started/what-is-exonum/)
   - [User guide](https://exonum.com/doc/get-started/java-binding)
-  - [How to run a Java service](exonum-java-binding-core/rust/ejb-app/TUTORIAL.md)
+  - [Cryptocurrency Example](exonum-java-binding-cryptocurrency-demo)
```

Comment:
Example cryptocurrency service :question:

Reply:
Cryptocurrency Example is better imo - more concise, but still understandable
Why don't we leave both [How to run a Java service] and [Cryptocurrency Example]?

Reply:
I thought it was less relevant than the user guide (which has such section) and an example service. There aren't too many items in this list though, so we can add it back.

Reply:
I'd suggest - 'Cryptocurrency' service example

Reply:
Shall we use ‘’ here?